explanation/jaas-plugin.md

```diff
@@ -0,0 +1,38 @@
+(jaas-plugin)=
+# `jaas` Plugin
+
+The JAAS ecosystem introduces the JIMM controller. A special controller that sits in front of your Juju controllers
+and acts as an authorisation gateway and aggregator.
+
+Interacting with the JIMM controller is done in the same fashion as Juju controllers, i.e. using the Juju CLI.
+JIMM also offers extra functionality, exposed by the `jaas` plugin.
+
+This document explains how to install the `jaas` plugin and how it works.
+
+## Installation
```

Comment:
I'd move this to a "Manage the `jaas` plugin how-to guide, with sections on Install, Upgrade, Uninstall.

Reply:
Yes.